src/Cake.SqlServer/SqlServerAliasesImpl.cs

```diff
@@ -167,5 +167,22 @@ internal static SqlConnection OpenSqlConnection(ICakeContext context, String con
                 throw;
             }
         }
+
+        internal static void SetCommandTimeout(ICakeContext context, int commandTimeout)
+        {
+            CommandTimeout = commandTimeout;
```

Comment:
Please add a log message to `context.Log.Debug($"Default SQL timeout have been changed to {commandTimeout}")`